pyston/filters/default_filters.py

```diff
@@ -210,14 +210,15 @@ class Filter(object):
     suffixes = {}
     choices = None
 
-    def __init__(self, identifiers_prefix, identifiers, identifiers_suffix, field=None, method=None):
+    def __init__(self, identifiers_prefix, identifiers, identifiers_suffix, model, field=None, method=None):
```

Comment:
Tak tady to vypadá že filtr je pouze na modely...

Reply:
ano je pouze na modely